pkg-builder.pl

```diff
@@ -10,11 +10,24 @@
 use File::Copy;
 use File::Path qw/make_path/;
 use Getopt::Long;
+use Getopt::Std;
```

Comment:
We already include Getopt::Long, why not use that and be consistent with argument handling?  Including both Long and Std seems like an unusual approach to say the least.